index.js

```diff
@@ -14,6 +14,8 @@ var dummyLogger = {
 };
 
 var defaultBehaviour = {
+  exchangeType: "topic",
+  exchangeOptions: {},
```

Comment:
We have the same functionality for queues, that prop is called "queueArguments". Perhaps we can name this "exchangeArguments" for consistency. ("exchangeOptions" is a better name per se, but I think consistency is more important here)

Reply:
Thanks, I saw that. Here we actually send in the whole options object though, the reason being that exp-amqp-connection does not use the other options (as opposed to the queue options). I am happy to change this to something like `{ arguments: exchangeArguments }` though, if you prefer.

Reply:
You are correct, the queue thing is a different case. Better to keep "exchangeOptions" then.